### ISSUE TITLE
Add support for InfluxDB

### ISSUE THREAD

**Issue by smilernet** [debrigand/Solaredge-monitoring#3]
Would it be possible to add InfluxDB as an option for the back-end rather than MySQL?

The serial for each optimizer can be used as a tag which then would enable the similiar functionality of the upload script to pvoutput i.e. per panel. For orientation statistics an addtional tag can be added to group panels of a similiar orientation.

InfluxDB as a time series database will be better for long term trend reporting; down sampling data with retention policies etc.

**Comment by smilernet** [comment]
No longer required as SolarEdge have disabled the local api in the latest firmware.

**Comment by ghost** [comment]
Indeed, a time series database would be better than plain old MySQL
